docs/geps/gep-04.rst

```diff
@@ -186,7 +186,35 @@ This is not satisfactory; ideally, functions would only be present in years wher
     - Users putting together their policy environment based on functions from different years
     - ...
 
+Additional functionalities
+--------------------------
 
+We implemented a small set of additional features that simplify the specification of
+certain types of functions of the tax and transfer system.
+
+Aggregation of variables
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Arguments of functions can also be aggregations of input variables or outputs of other
+functions on the tax unit or household level. In the example above, one argument of the
+function ``soli_st_tu`` is ``abgelt_st_tu``. This variable would be typically specified
```

Comment:
I think a real-life example would be better here. I do not expect people to read through this and keep in mind the precise example from a few paragraphs ago, just in order to re-use it with a hypothetical case? (`abgelt_st_tu` is not fully obvious because of the exemption amount, which could be on individual or joint accounts, I guess).